tests/test_object_manager.py

```diff
@@ -365,8 +366,26 @@ def test_find_hostname(self):
             extattrs=None, force_proxy=mock.ANY, return_fields=mock.ANY,
             max_results=None)
 
+        connector.get_object.reset_mock()
```

Comment:
I think L369-L376 should be moved to independent test case with name like:
test_find_hostname_by_network_view_and_dns_view()
then L378-L385 should be also separate test:
test_find_hostname_by_network_view()
And to be consistent L351 has to be renamed to:
def test_find_hostname_by_dns_view()